pack/ptc/tuo_encounter.json

```diff
@@ -387,7 +387,7 @@
         "shroud": 2,
         "subname": "Daniel's Cell",
         "text": "<b>Forced</b> - After Patient Confinement is revealed: Advance to act 2b.",
-        "traits": "Arkham Asylum.",
+        "traits": "",
```

Comment:
So I'm not positive, but I think the way the update script works, we needs an actual value for it to be cleared out, since it previously had a value omitting it / setting it to null would result in it not being changed.